draft-ietf-rats-corim.md

```diff
@@ -2448,8 +2449,27 @@ The comparison MUST return false if there are no hash algorithms from the condit
 
 ##### Comparison for raw-value entries
 
+A `raw-value` entry contains binary data whose meaning the Verifier is not required to understand.
```

Comment:
"Not required" is different than MUST treat opaquely, or only with mask comparison. We don't want this type to have different meanings in different verifiers.

Reply:
I changed the text to say that a Verifier following this spec must treat `raw-value` as opaque data.